# AE 01: Meet the penguins
## Greg Chism

For this application exercise, we'll use the pandas and palmerpenguins packages.

In [1]:
import pandas as pd
from palmerpenguins import load_penguins

First, let's read in the data.

In [2]:
penguins = load_penguins()

The dataset we will visualize is called `penguins`.
Let's `.info()` at it.

In [3]:
2 * 4

8

-   **Your turn:** Replace `#add code here` with the code for "info"ing at the data `penguins` data frame – `penguins.info()`. Render the document and view the output.

In [4]:
# add code here

-   **Demo:** First, replace the blank below with the number of rows in the `penguins` data frame based on the output of the chunk below. Then, replace it with "inline code" and render again.

In [5]:
len(penguins)

344

There are 344 penguins in the `penguins` data frame.